text/0000-undisambiguated-generics.md

```diff
@@ -69,29 +69,43 @@ a < b > ( c ); // error: chained comparison operators require parentheses
 ```
 
 This chained comparison syntax is therefore no longer ambiguous. There are, however, two cases in
-which the syntax is currently ambiguous (arguably these are a single case, correspoding to the same
-ambiguity with `<` and `<<` respectively).
+which the syntax is currently ambiguous.
 
+First:
 ```rust
 // The following:
-(a < b, c > (d));
+(a < b, c > (d))
 // Could be a generic function call...
-( a<b, c>(d) );
+( a<b, c>(d) )
 // Or a pair of comparisons...
-(a < b, c > (d));
+(a < b, c > (d))
 
-// The following:
-(a << B as C > ::D, E < F >> (g));
+// This is true with both `<` and `<<`:
+(a << B as C > ::D, E < F >> (g))
 // Could be a generic function call (with two arguments)...
-( a<<B as C>::D, E<F>>(g) );
+( a<<B as C>::D, E<F>>(g) )
 // Or a pair of bit-shifted comparisons...
-(a << B as C > ::D, E < F >> (g));
+(a << B as C > ::D, E < F >> (g))
+```
+
+Second:
+```rust
+// The following:
+a < b >> c
+// Could be a comparison of a generic expression...
+a<b> > c
+// Or a bit-shift followed by a comparison...
+a < b >> c
 ```
 
 Ultimately, these cases do not seem occur naturally in Rust code. A
 [Crater run on over 20,000 crates](https://github.com/rust-lang/rust/pull/53578#issuecomment-421475443)
 determined that no crates regress if the ambiguity is resolved in favour of a generic expression
-rather than tuples of comparisons of this form. We propose that resolving this ambiguity in favour
+rather than tuples of comparisons of this form. However, there are some occurrences of syntax
+similar to the second ambiguity ([1](https://sourcegraph.com/github.com/dropbox/rust-brotli/-/blob/src/enc/backward_references.rs#L1257:32),
+[2](https://sourcegraph.com/github.com/dropbox/rust-brotli/-/blob/src/enc/encode.rs#L1905:46)).
```

Comment:
I suggest making these to be permalink to the GitHub repository:

1. https://github.com/dropbox/rust-brotli/blob/ce7b3618f9df942e9340bf3e767b2d3e3caea4b3/src/enc/backward_references.rs#L1257
2. https://github.com/dropbox/rust-brotli/blob/ce7b3618f9df942e9340bf3e767b2d3e3caea4b3/src/enc/encode.rs#L1905 (but this one is placed behind a comment, the uncommented one is https://github.com/dropbox/rust-brotli/blob/ce7b3618f9df942e9340bf3e767b2d3e3caea4b3/src/enc/encode.rs#L2842)